packages/manager/modules/pci/src/projects/project/ai-dashboard/style.scss

```diff
@@ -0,0 +1,63 @@
+ovh-manager-pci-project-a-i-dashboard {
```

Comment:
```suggestion
ovh-manager-pci-project-ai-dashboard {
```

Reply:
Changing this would not work, as it is the component's name: 
![image](https://user-images.githubusercontent.com/24541481/213704960-0f889f08-0459-4f7c-ad19-6383ecd48211.png)